typescript/sdk/src/hardhat/TestCoreApp.ts

```diff
@@ -1,18 +1,18 @@
+import { ethers } from 'ethers';
```

Comment:
Should the folder even be called hardhat? As far as I can tell there is nothing hardhat specific in here right?

Reply:
I was worried `test` would make it seem like tests, not test utilities. Any name suggestions?

Reply:
With the test now colocated, there is some hardhat specific stuff

Reply:
I would be fine to just move it to the `core` folder? I still don't see what the hardhat specific code in here is?